### ISSUE TITLE
Add telemetry for load_screen_fails_and_caregivers_into_screen_follow_up.R

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#61]
Add telemetry for `load_screen_fails_and_caregivers_into_screen_follow_up.R`

The report needs to enumerate

```
nrow(rc_upload)
result$records_affected_count
result$success
```

To get those last two values, make this change at https://github.com/ctsit/tt_engine/blob/6af6d5806d25c9e1207e5678f50bd87ee34e2bd3/load_screen_fails_and_caregivers_into_screen_follow_up.R#L66

```
result <- redcap_write(rc_upload,
        redcap_uri = Sys.getenv("URI"),
        token = Sys.getenv("SCREEN_TOKEN"))
```

The subject line of the report should be 

```
email_subject <- paste("Screen fails loaded", Sys.getenv("INSTANCE"), script_run_time, "| Success", write_data$success)
```

Add a line like this to `examples/tt_engine_prod.cron`

```
/usr/bin/docker run -v /ctsit/tt_engine/oneuftesting_ctsit.key:/root/.ssh/id_rsa \
 --rm --env-file /ctsit/tt_engine/prod.env \
tt_engine Rscript load_screen_fails_and_caregivers_into_screen_follow_up.R
```

This job should run at 1205 and 1705.

**Comment by ChemiKyle** [comment]
Overloading `--env-file` does not appear to work as intended; could be due to `dotenv` only loading `.env` when called or something inherent to docker using `.env` for its own purposes. I can use `load_dot_env` with a cli arg and a volume, but it seems easier to just add a `RESULT_DIGEST_EMAIL` field with additional recipients to `.env`.

**Comment by pbchase** [comment]
Sure.  That's fine.

**Comment by ChemiKyle** [comment]
Uh, nevermind. I'm too used to having code mounted as a volume, I never rebuilt.

**Comment by pbchase** [comment]
addressed by PR #63
